modules/bbp-core/src/main/resources/schemas/bbp/core/brainLocation/v0.1.0.json

```diff
@@ -0,0 +1,159 @@
+    {
+      "bbpvect3dsh": "{{base}}/schemas/bbp/core/vector3d/v0.1.0/shapes/",
+      "bbptlotsh": "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+      "bbpqsh": "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/core/brainlocation/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+                "{{base}}/schemas/bbp/core/vector3d/v0.1.0",
+                "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0"
+  ],
+  "@type": "owl:Ontology",
+  "shapes": [
+    {
+      "@id": "this:BrainLocationShape",
+      "@type": "sh:NodeShape",
+      "label": "Brain location shape definition",
+      "property": [
+        {
+          "path": "nsg:brainRegion",
+          "name": "Brain region",
+          "description": "Brain region",
+          "node": "bbptlotsh:BrainRegionOntologyTermShape",
+          "minCount": 1
+        },
+        {
+          "path": "nsg:coordinatesInBrain",
+          "name": "Coordinates in brain",
+          "description": "Coordinates in brain",
+          "node": "this:CoordinatesInBrainShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:coordinatesInSlice",
+          "name": "Coordinates in slice",
+          "description": "Coordinates in slice",
+          "node": "this:CoordinatesInSliceShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:subRegion",
+          "name": "Sub region",
+          "description": "Sub region (e.g. CA3b)",
+          "node": "bbptlotsh:BrainRegionOntologyTermShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:positionInLayer",
+          "name": "Position in layer",
+          "description": "Position in layer (deep / superficial)",
+          "sh:in" : {
+            "@list" : [ "Deep", "Superficial" ]
+          },
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:longitudinalAxis",
+          "name": "Longitudinal axis (dorsal / ventral)",
+          "sh:in" : {
+            "@list" : [ "Dorsal", "Ventral" ]
+          },
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:distanceToBoundary",
+          "name": "Distance to boundary",
+          "description": "Distance to boundary",
+          "node": "this:DistanceToBoundaryShape",
+          "maxCount": 1
+        }
+      ]
+    },
+    {
+      "@id": "this:CoordinatesInBrainShape",
+      "@type": "sh:NodeShape",
+      "label": "Coordinates in brain shape definition",
+      "and": [
+        {
+          "node": "bbpvect3dsh:Vector3DShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:spatialReferenceSystem",
+              "name": "Spatial Reference System",
+              "class": "nsg:SpatialReferenceSystem",
+              "minCount": 1,
+              "maxCount": 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id": "this:CoordinatesInSliceShape",
```

Comment:
Still think one shape for coordinates is better for both slice and atlas. Let merge this one and have a look again later.